packages/react-composites/src/composites/common/utils.tsx

```diff
@@ -53,3 +53,12 @@ export const useMinMaxDragPosition = (modalLayerHostId: string, rtl?: boolean):
  * @private
  */
 export const defaultSpokenLanguage = 'en-us';
+
+/**
+ * @private
+ *  * Generate a unique id
+ * TODO: Replace with useId() once React 18 becomes a required dependency.
+ */
+export const generateUniqueId = (): string => {
```

Comment:
are we placing this hook in both files with the hope to replace them some day? should we not put this in the `acs-ui-common` package so we can use these in other places too if need be?

Reply:
Good catch, yeah that's right. I moved it to acs-ui-common now! :D